test/ethclient.go

```diff
@@ -1633,6 +1635,11 @@ func (c *Client) AuctionEventInit(genesisBlockNum int64) (*eth.AuctionEventIniti
 	}, 1, nil
 }
 
+// GetCoordinatorsLibP2PAddrs required for the interface
+func (c *Client) GetCoordinatorsLibP2PAddrs() ([]multiaddr.Multiaddr, error) {
+	return nil, errors.New("TODO")
```

Comment:
What does this error mean??

Reply:
This is required because of how the tests are implemented: using an interface over the entire package. Since the tests for the libp2p stuff are tested in another way, there is no need to implement this on the testing interface, however it's required to fulfill the interface